uhu/core/objects.py

```diff
@@ -52,10 +52,22 @@ def load(self, callback=None):
             obj.load(callback=callback)
         call(callback, 'finish_objects_load')
 
+    def _check_duplicate_object_entry(self, options):
+        for obj in self.objects:
+            metadata_list = [entry.to_metadata() for entry in obj]
+            equal = True
+            for metadata in metadata_list:
+                if metadata['filename'] != options['filename'] or metadata['mode'] != options['mode']:
+                    equal = False
```

Comment:
and compare *if* equal.

Something as:
```python
    for m in metadata_list:
        if m['filename'] == options['filename'] and \
           m['mode'] == options['mode']:
            duplicated = True
            break
```

Reply:
Here I'm comparing a piece of a pair at a time, and supposing the pair is a list of arbitrary length to avoid direct indexing. Your suggestion tests if one of the files is the same, and I though they ought to be both the same.

I used `or` because I understood that the same mode could not show up twice (in different pairs).